src/set_theory/ordinal/exponential.lean

```diff
@@ -10,8 +10,9 @@ import set_theory.ordinal.arithmetic
 > THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
 > Any changes to this file require a corresponding PR to mathlib4.
 
-In this file we define the power function and the logarithm function on ordinals,
-
+In this file we define the power function and the logarithm function on ordinals. The two are
+related by the lemma `ordinal.lt_opow_iff_log_lt : x < (b^c) ↔ log b x < c` for nontrivial inputs 
```

Comment:
Isn't the ≤ version a stronger characterization?

Reply:
They're entirely equivalent, but to be fair we do prove the `≤` version before.